.gitignore

```diff
@@ -21,3 +21,9 @@
 *,cover
 .venv
 .env
+poetry.lock
```

Comment:
poetry.lock needs to be checked in, this is kind-of the point of poetry... to make sure everyone uses the same level of dependencies.